js/binance.js

```diff
@@ -4937,4 +4938,53 @@ module.exports = class binance extends Exchange {
             'info': response,
         };
     }
+
+    async fetchBorrowRateHistory (code, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (limit === undefined) {
+            limit = 93;
+        } else if (limit > 93) {
+            // Binance API says the limit is 100, but illegal parameter errors are returned when limit is > 93
+            throw new BadRequest (this.id + ' fetchBorrowRateHistory limit parameter cannot exceed 93');
+        }
+        const currency = this.currency (code);
+        const request = {
+            'asset': currency['id'],
+            'limit': limit,
+            'startTime': since,
+        };
+        if (since) {
```

Comment:
We want to refrain from implicit type coersion and always be explicit in such cases:

```JavaScript
if (since !== undefined) {
}
```